### PR TITLE
Mark 'venv sync' as deprecated

### PR DESCRIPTION
**Pull request by connesy**
Part of https://github.com/SallingGroup-AI-and-ML/venv-cli/issues/13

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased](https://github.com/SallingGroup-AI-and-ML/venv-cli/tree/develop)
 
+## Minor changes
+* `venv sync` command marked as deprecated with removal planned for `v2.0`. Use `venv install <requirments>.lock` instead. [#14](https://github.com/SallingGroup-AI-and-ML/venv-cli/pull/14)
```
Comment (rasmusoej) [inline CHANGELOG.md:6]:
```suggestion
* `venv sync` command marked as deprecated with removal planned for `v2.0`. Use `venv install <requirements>.lock` instead. [#14](https://github.com/SallingGroup-AI-and-ML/venv-cli/pull/14)
```

**Review by rasmusoej — APPROVED** [review]
Would it make sense to keep the `venv sync` command, but just have it as an alias for `venv install <requirements>.lock`?

**Comment by connesy** [comment]
> Would it make sense to keep the `venv sync` command, but just have it as an alias for `venv install <requirements>.lock`?

That would still require maintaining the command with `help` text, documentation, tests, functionality to make sure only `.lock` files were passed, and tab completions. 
To keep the tool minimal, I'd rather advise users to create an alias themselves if this is something they do often.

File: CHANGELOG.md (hunk 1 above)
Comment (connesy) [reply CHANGELOG.md]: Fixed in https://github.com/SallingGroup-AI-and-ML/venv-cli/commit/8665832dbee2f165127bf79150ebe1b5362312b8

**Comment by rasmusoej** [comment]
> > Would it make sense to keep the `venv sync` command, but just have it as an alias for `venv install <requirements>.lock`?
> 
> That would still require maintaining the command with `help` text, documentation, tests, functionality to make sure only `.lock` files were passed, and tab completions. To keep the tool minimal, I'd rather advise users to create an alias themselves if this is something they do often.

Fair :)

**Pull request merged**